README.md

```diff
@@ -36,8 +36,8 @@ For running the script, you will need Python version > 3.7. You can download/ins
 
 Once python is downloaded and installed, you can verify the version from your terminal/command window by typing `python --version`.
 
-To run the script, you should switch to the FreeRTOS-Plus-TCP directory that was created using the [Cloning this repository](#cloning-this-repository) step above.
```

Comment:
Should this have been removed?

Reply:
Yeah, those instructions no longer exist in this README